SDL/Module.cuh

```diff
@@ -95,11 +92,11 @@ namespace SDL
         ModuleType* moduleType;
         ModuleLayerType* moduleLayerType;
        
-        ALPAKA_FN_HOST_ACC ModuleType parseModuleType(unsigned int index);
-        ALPAKA_FN_HOST_ACC ModuleType parseModuleType(short subdet, short layer, short ring);
-        ALPAKA_FN_HOST_ACC unsigned int parsePartnerModuleId(unsigned int detId, bool isLowerx, bool isInvertedx);
-        ALPAKA_FN_HOST_ACC ModuleLayerType parseModuleLayerType(unsigned int index);
-        ALPAKA_FN_HOST_ACC ModuleLayerType parseModuleLayerType(ModuleType moduleType, bool isInvertedx, bool isLowerx);
+        ModuleType parseModuleType(unsigned int index);
+        ModuleType parseModuleType(short subdet, short layer, short ring);
+        unsigned int parsePartnerModuleId(unsigned int detId, bool isLowerx, bool isInvertedx);
+        ModuleLayerType parseModuleLayerType(unsigned int index);
+        ModuleLayerType parseModuleLayerType(ModuleType moduleType, bool isInvertedx, bool isLowerx);
```

Comment:
Are we going a bit back and forth here? Why was the alpaka keywords, supposedly added in one of your previous PRs, removed here?

Reply:
This was carried over from master, where these functions have the hostdev prefix, but I realized these functions are only ever run on the host, so their prefixes are kind of pointless. Also, I realized a lot of these functions aren't even being used so I just removed most of them.